internal/handler/products/qparamkey.go

```diff
@@ -1,7 +1,8 @@
-package products
-
-// defines query param keys for this route
-
-const QUERY_LIMIT = "limit"
-const QUERY_OFFSET = "offset"
-const QUERY_SEARCH = "search"
+package products
+
+// defines query param keys for this route
+
+const QUERY_LIMIT = "limit"
+const QUERY_OFFSET = "offset"
+const QUERY_SEARCH = "search"
+const QUERY_BRAND = "name"
```

Comment:
Please be consistent on naming queries. I suggest you to change `QUERY_BRAND` to `QUERY_NAME` to match the value. Or you might want to change the value to `"brand"`.

Reply:
copy that